packages/collector/test/tracing/cloud/azure/blob/app.js

```diff
@@ -0,0 +1,343 @@
+/*
+ * (c) Copyright IBM Corp. 2021
+ * (c) Copyright Instana Inc. and contributors 2018
+ */
+
+/* eslint-disable no-console */
+
+'use strict';
+
+// const agentPort = process.env.INSTANA_AGENT_PORT;
+
+require('../../../../..')();
+
+const { BlobServiceClient, BlobBatchClient, BlobClient, StorageSharedKeyCredential } = require('@azure/storage-blob');
+
+const express = require('express');
+// const morgan = require('morgan');
+// const request = require('request-promise-native');
+const bodyParser = require('body-parser');
+const port = require('../../../../test_util/app-port')();
+
+const app = express();
+const logPrefix = `Express / azure blob App (${process.pid}):\t`;
+const fs = require('fs');
+
+const filePath = `${__dirname}/sample.pdf`;
+const localFilePath = `${__dirname}/out.pdf`;
+const data1 = fs.readFileSync(filePath);
+const accountKey = process.env.ACC_KEY;
+const connStr = process.env.CONN_STR;
+const storageAccount = process.env.STORAGE_ACC;
+
+const blobServiceClient = BlobServiceClient.fromConnectionString(connStr);
+const containerName = process.env.CONTAINER_NAME;
+const blobName = 'first.pdf';
+const containerClient = blobServiceClient.getContainerClient(
+  containerName
+);
+const cred = new StorageSharedKeyCredential(storageAccount, accountKey);
+const data = Buffer.from(data1.toString('base64'), 'base64');
+const blockBlobClient = containerClient.getBlockBlobClient(blobName);
+
+const uploadDocumentToAzure = async (options = {}) => {
+  try {
+    const opts = {
+      blobHTTPHeaders: {
+        blobContentType: 'application/pdf'
+      }
+    };
+    if (options.maxSingleShotSize) {
+      opts.maxSingleShotSize = options.maxSingleShotSize;
+    }
+    const response = await blockBlobClient.uploadData(data, opts);
+    if (response._response.status !== 201) {
+      throw new Error(
+        `Error uploading document ${blockBlobClient.name} to container ${blockBlobClient.containerName}`
+      );
+    } else {
+      console.log('uploaded successsfully');
+      return ('uploaded successsfully');
+    }
+  } catch (e) {
+    console.log('err:', e);
+  }
+};
+
+const deleteDocumentFromAzure = async (_blobName) => {
+  const response = await containerClient.deleteBlob(_blobName);
+  if (response._response.status === 202) {
+    console.log(`Deleted ${_blobName}`);
+  }
+  if (response._response.status !== 202) {
+    throw new Error(`Error deleting ${_blobName}`);
+  }
+  return response;
+};
+
+const download = async (_blobName) => {
+  const _blockBlobClient = containerClient.getBlockBlobClient(_blobName);
+  try {
+    const downloadBlobResponse = await _blockBlobClient.downloadToFile(localFilePath);
+    console.log(`Downloaded blob to ${localFilePath}:`);
```

Comment:
Please reduce the number of console.* in all files 👍

Reply:
removed unwanted logs